src/mlpack/tests/binarize_test.cpp

```diff
@@ -13,16 +13,14 @@
 #include <mlpack/core/data/binarize.hpp>
 #include <mlpack/core/math/random.hpp>
 
-#include <boost/test/unit_test.hpp>
-#include "test_tools.hpp"
+#include "test_catch_tools.hpp"
+#include "catch.hpp"
 
 using namespace mlpack;
 using namespace arma;
 using namespace mlpack::data;
 
-BOOST_AUTO_TEST_SUITE(BinarizeTest);
-
-BOOST_AUTO_TEST_CASE(BinerizeOneDimension)
+TEST_CASE("BinerizeOneDimension", "[BinarizeTest]")
```

Comment:
```suggestion
TEST_CASE("BinarizeOneDimension", "[BinarizeTest]")
```
I know this was not your error but I saw it and thought it would be easy to fix while we're here. :)